expression/builtin_string_vec_test.go

```diff
@@ -182,6 +184,9 @@ var vecBuiltinStringCases = map[string][]vecExprBenchCase{
 	ast.Reverse: {
 		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETString}, geners: []dataGenerator{&randLenStrGener{10, 20}}},
 		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETString}, geners: []dataGenerator{&defaultGener{0.2, types.ETString}}},
+		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETString},
+			childrenFieldTypes: []*types.FieldType{{Tp: mysql.TypeString, Flag: mysql.BinaryFlag, Collate: charset.CollationBin}},
```

Comment:
For curiosity, why do we need to state `childrenFieldTypes` specifically here?

Reply:
If we don't set the `Collate` to `CollationBin`, the function `builtinReverseSig` will be used instead of `builtinReverseBinarySig`.